patches/fixGoingBack.js

```diff
@@ -0,0 +1,74 @@
+import { waitForRender } from './apis/waitForElement.js';
+import { setHighlights } from './newMobileNavbar/highlights.js';
+
+const fixGoingBack = async () => {
+    addEventListener('popstate', (e) => {
+        if (e.state?.more) {
+            document.querySelector('.more-popup').style.display = "block";
+        } else {
+            document.querySelectorAll('.list-modal').forEach((e) => {
+                e.style.display = "none";
+            });
+        }
+        if (document.querySelector('.modal-user').classList.contains('active')) {
+            history.forward();
+            document.querySelector('.modal-user').classList.remove('active');
+            document.querySelector('.modal-background').classList.remove('active');
+        }
+        if (!location.hostname.includes('wiadomosci')) {
```

Comment:
Correct place for this code is in the patch related to that.

This should not be implemented based on does someone have this patch enabled

Reply:
To be exact it works on a first modal but not the 2nd (adding the id data to the path should prob fix that as those would be different paths)